code/modules/reagents/reagent_dispenser.dm

```diff
@@ -329,3 +335,27 @@
 		to_chat(user, "<span class='notice'>Sprayer refilled.</span>")
 		playsound(get_turf(src), 'sound/effects/refill.ogg', 50, 1, -6)
 		return 1
+		
+/obj/structure/reagent_dispensers/proc/wrenchAnchor(var/mob/user) //proc to wrench a wrenchable dispenser into place
```

Comment:
@JustSumBody yeah, remove (or alter it so machines still power off and such, even while calling the parent) the machinery level proc.

Not need to move the `wrenchable` variable.